numba/tests/test_builtins.py

```diff
@@ -805,18 +793,6 @@ def test_oct_npm(self):
         with self.assertTypingError():
             self.test_oct(flags=no_pyobj_flags)
 
-    def test_ord(self, flags=enable_pyobj_flags):
```

Comment:
Removing this also removes all references to the `ord_usecase` defined above. So it may well have to be removed also.

Reply:
done